docs/dev/uap/key_concepts.rst

```diff
@@ -0,0 +1,106 @@
+Key Concepts
+============
+
+This document is based on `preliminary discussions in our Google docs <https://docs.google.com/document/d/14H3Rc922-IYy6DIEV76_3hXCQPwl6OmKecJkAheTkI4/edit#heading=h.tthq28g2awwq>`_.
+It describes the key concepts found in the core auth module, and how they touch on other key concepts from other
+modules.
+
+**Kolibri** is a platform with two major purposes:
+#. Serve educational content in "disconnected" scenarios, and support students and teachers with self-paced,
+peer-interactive, and blended learning practices.
+#. Collect information about users and their interaction with that content, and allow it to be synced to other devices
+or to a central repository.
+
+The Kolibri platform has two major software components. The **Distributed Application** (which may be referred to as
+"Kolibri" or the "App") is a software application which serves content (like videos and exercises), stores user data
+to a local database. A Distributed Application is installed and runs on a **Device** (a physical machine such as a
+laptop or tablet). The Distributed Application can store data for one or more **Facilities** (corresponding to a
```

Comment:
Should I put this comment in the GDoc? I hoped that we could come up with a better concept since the KA Lite days of Facility. I still don't think it's entirely intuitive whether it's a whole school or a classroom within a school. Could it be better to call it a Hub? Or a Deployment? A LocalNetwork? A Cluster? In case there's already been a brainstorm or this feedback is totally mistimed, please disregard :)